src/pages/instances/forms/InstanceFormMenu.tsx

```diff
@@ -104,6 +105,7 @@ const InstanceFormMenu: FC<Props> = ({
               <MenuItem label={RESOURCE_LIMITS} {...menuItemProps} />
               <MenuItem label={SECURITY_POLICIES} {...menuItemProps} />
               <MenuItem label={SNAPSHOTS} {...menuItemProps} />
+              <MenuItem label={MIGRATION} {...menuItemProps} />
```

Comment:
should we hide this menu item if we are creating a container?

Reply:
I was thinking about this, but then didn't want to make the menu options disappear while configuring.

Reply:
yeah you are right, I think having the setting disabled makes more sense then :+1: